### DISCUSSION TITLE
which one is  better : a MinIO pod with six 1TB PVCs or one 6TB PVC,

### DISCUSSION THREAD

**Discussion by adababys** [minio/minio#20027] [Q&A] [answered]
When deploying a 14-node MinIO cluster in a Kubernetes environment, there are two options for provisioning storage to a MinIO pod on one of the nodes: allocating six 1TB PVCs or allocating a single 6TB PVC. The PVCs are provided by a Kubernetes StorageClass. Which deployment method is better?

**Comment by klauspost** [comment, marked as answer]
MinIO prefers storage that matches physical disks. Read more at https://min.io/docs/minio/kubernetes/upstream/operations/concepts.html

Your underlying storage also defines your true availability. [Reach out](https://min.io/pricing) if you want a detailed review of your system that reaches beyond the documentation.

**Reply by adababys** [reply]
Given the current state of storage, would you recommend allocating one 6TB PVC or six 1TB PVCs for a MinIO pod?

**Reply by harshavardhana** [reply]
[Reach out](https://min.io/pricing) if you want a detailed review of your system that reaches beyond the documentation.
